src/datagrid/shortcuts.ts

```diff
@@ -0,0 +1,26 @@
+import type { Shortcut } from '@bambooapp/bamboo-molecules/shortcuts-manager';
+import { isMac } from '@bambooapp/bamboo-molecules';
+
+export const shortcuts: Shortcut[] = [
+    {
+        name: 'move-cell-focus',
+        keys: [
```

Comment:
can we accept `(string | string[])[]` and normalise it with code?

Reply:
done